.github/workflows/ci.yml

```diff
@@ -19,21 +19,24 @@ jobs:
   core:
     strategy:
       matrix:
-        os: [ubuntu-20.04, ubuntu-latest, macos-latest]
+        os: [ubuntu-20.04, ubuntu-22.04]
```

Comment:
use only ubuntu-latest, unless you have good reason to keep the oldies here

Reply:
Per Discord, this one I am not sure about. There is a separate issue #5 that covers aligning Iggy server and SDK test matrices and can revisit as part of that.